packages/web-app-files/src/composables/resourcesViewDefaults/useResourcesViewDefaults.ts

```diff
@@ -76,6 +69,14 @@ export const useResourcesViewDefaults = <T, TT, TU extends any[]>(
   const currentTilesSize = computed((): string => String(currentTilesSizeQuery.value))
   const viewSize = useViewSize(currentTilesSize)
 
+  const fields = computed((): SortField[] => {
```

Comment:
Why not call it `sortFields`? 😁

Reply:
in the context of `useSort` the name `fields` is clear, but outside of that composable it's not IMO.

Reply:
Because it was named `fields` before 🙈  I need to check where we use this, maybe I can easily rename it without changing it all over the place.

Reply:
Seems that only a few locations were affected :)